src/streams/WritableStream.ts

```diff
@@ -104,7 +112,7 @@ export default class WritableStream<T> {
 	protected _queue: SizeQueue<Record<T>>;
 	protected _writing: boolean;
 
-	constructor(underlyingSink: Sink<T>, strategy: Strategy<T> = {}) {
+	constructor(underlyingSink: Sink<T> = {}, strategy: Strategy<T> = {}) {
```

Comment:
Line 210 sets the sink to undefined to release the WritableStream's reference to the sink. At that point it no longer matters what sink is set to - calling any method will fail due to checks for `state === State.Closed`